### DISCUSSION TITLE
Any plans to support DNS over HTTPS?

### DISCUSSION THREAD

**Discussion by JoelDKraft** [MichaCo/DnsClient.NET#204] [Ideas]
I have an application where I have to use DoH in addition to what I do with DnsClient.NET.  It would be great if it could all be done using the same library!

I cheat and use the Google's JSON endpoint so I don't have to deal with the gory details. My understanding is that the raw the messages are exactly the same as the UDP and TCP transports, though.

**Comment by MichaConrad** [comment]
Hi Joel, 
Yeah there was a discussion about this feature here #98. 
The problem with http transport for me is that I'd have to handle the HttpClient lifetime somehow. I did not had time to really settle on a good solution and implement it, but, yeah, I might add it at some point.
